ssl/t1_trce.c

```diff
@@ -665,7 +665,8 @@ static int ssl_print_random(BIO *bio, int indent,
 
     if (*pmsglen < 32)
         return 0;
-    tm = (p[0] << 24) | (p[1] << 16) | (p[2] << 8) | p[3];
+    tm = ((unsigned int)p[0] << 24) | ((unsigned int)p[1] << 16)
+         | ((unsigned int)p[2] << 8) | (unsigned int)p[3];
```

Comment:
Please do that in another PR.
Reason: there is no test coverage for the undefined behavior.
so no test gets fixed by this.  please correct me if I am wrong.

Reply:
I disagree with doing a different PR just for this reason.

Reply:
yeah we disagree.